src/styles/index.scss

```diff
@@ -2,2 +1,137 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  width: 200px;
+  height: 408px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+
+  &__image {
+    margin-top: 33px;
+
+    &-photo {
+      width: 160px;
+      height: 134px;
```

Comment:
Using fixed width and height for images can cause distortion or cropping if the image's aspect ratio doesn't match. Consider using properties like max-width, max-height, and object-fit to control the image size while preserving its aspect ratio.